package.json

```diff
@@ -112,9 +112,9 @@
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.2.0",
     "husky": "7.0.4",
-    "jest": "29.4.3",
-    "jest-axe": "5.0.1",
-    "jest-environment-jsdom": "29.5.0",
+    "jest": "29.7.0",
+    "jest-axe": "9.0.0",
```

Comment:
This is a big jump.  Are there any concerns?

Reply:
I had some concerns regarding the WCAG standard being used and our tests breaking but no, it was seamless. We should update this though because this updates the min node version to 16.